boards/xtensa/esp32s2_lolin_mini/doc/index.rst

```diff
@@ -0,0 +1,92 @@
+.. _esp32s2_lolin_mini:
+
+ESP32-S2
```

Comment:
Maybe change the name to "esp32s2 lolin mini"

Reply:
Done